{"uuid":"21696013-6037-43a6-8372-3f306c7cade3","historyId":"b9f36b498a5621238bae6a4667032db7","fullName":"tests.SignupTest.invalidEmail_ShowsValidation","labels":[{"name":"package","value":"tests.SignupTest"},{"name":"testClass","value":"tests.SignupTest"},{"name":"testMethod","value":"invalidEmail_ShowsValidation"},{"name":"parentSuite","value":"Playwright Suite"},{"name":"suite","value":"UI Tests"},{"name":"subSuite","value":"tests.SignupTest"},{"name":"host","value":"192.168.1.109"},{"name":"thread","value":"90196@192.168.1.109.TestNG-test-UI Tests-1(49)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Invalid email format shows validation","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Invalid emailsss message should be visible expected [true] but found [false]","trace":"java.lang.AssertionError: Invalid emailsss message should be visible expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat tests.SignupTest.invalidEmail_ShowsValidation(SignupTest.java:48)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:328)\n\tat org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)\n\tat java.base/java.lang.Thread.run(Thread.java:1447)\n"},"stage":"finished","steps":[],"attachments":[{"name":"Failure screenshot","source":"9750fb4b-c5aa-4d0f-8434-dbc53339522f-attachment.png","type":"image/png"}],"parameters":[],"start":1755730488401,"stop":1755730489630}